nautobot_golden_config/utilities/db_management.py

```diff
@@ -0,0 +1,15 @@
+"""Functions to manage DB related tasks."""
+from django.db import connections
+
+
+def close_threaded_db_connections(func):
+    """Decorator that clears idle DB connections in thread."""
+
+    def inner(*args, **kwargs):
+        """Inner function."""
+        func(*args, **kwargs)
+
+        for c in connections.all():
```

Comment:
```suggestion
        for connection in connections.all():
```
Not tested, suggestion just to satisfy pylint.